src/providers/GaugeProvider.tsx

```diff
@@ -17,28 +24,29 @@ export default function GaugeProvider({
   const gaugeControllerContract = useGaugeControllerContract()
   const basicPools = useContext(BasicPoolsContext)
   const gaugeMinterContract = useGaugeMinterContract() // only exists on mainnet
-  const [gauges, setGauges] = useState<Gauges>(initialGaugesState)
+  const [gauges, setGauges] = useState<LPTokenAddressToGauge>({})
+  const [gaugeCount, setGaugeCount] = useState(0)
```

Comment:
Is this common practice even if `gauges` default state is `{}` and not `0`?

Reply:
It is not common practice, just want to split gauge and gaugeCount and insert loading.
But we are using ReactQuery to handle loading so I removed it.